### PR TITLE
Expanded courses

### PR DESCRIPTION
**Pull request by Erin007**
Students and teachers will be shown up to 4 of their most recent courses as CourseCards on their homepage (students will actually see 5 courses including a TopCourse in upcoming work). If they have more than 4 courses, there will be a View More button, that, when clicked, reveals the rest of their courses. 

more courses

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @tanyaparker

File: apps/src/templates/studioHomepages/SeeMoreCourses.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import i18n from "@cdo/locale";
+import CourseCard from './CourseCard';
+import ContentContainer from './ContentContainer';
+import ProgressButton from "../progress/ProgressButton";
+import shapes from './shapes';
+
+const SeeMoreCourses = React.createClass({
```
Comment (Bjvanminnen) [inline apps/src/templates/studioHomepages/SeeMoreCourses.jsx]: Might be worth a short description describing what this component does.

File: apps/src/templates/studioHomepages/RecentCourses.jsx [hunk 2]
```diff
@@ -17,50 +18,54 @@ const RecentCourses = React.createClass({
 
   render() {
     const { courses, showAllCoursesLink, isTeacher, heading, isRtl } = this.props;
+    const topFourCourses = courses.length > 4 ? courses.slice(0,4) : courses;
```
Comment (breville) [inline apps/src/templates/studioHomepages/RecentCourses.jsx:21]: Is 4 a magic number that should be extracted to a named constant along with a comment about what it means?

**Comment by breville** [comment]
The screen capture has a grey panel.. is that an artifact?

**Comment by tanyaparker** [comment]
I believe the gray panel is from storybook. For whatever reason the white background doesn't extend to the same full width as the components.

**Pull request merged**